iron-a11y-announcer.js

```diff
@@ -107,5 +113,5 @@ IronA11yAnnouncer.requestAvailability = function() {
     IronA11yAnnouncer.instance = document.createElement('iron-a11y-announcer');
   }
 
-  document.head.appendChild(IronA11yAnnouncer.instance);
+  document.body.appendChild(IronA11yAnnouncer.instance);
```

Comment:
What happens if this is called before document.body exists? I seem to recall we've had at least one bug like that before

Reply:
TIL. That's a good point. I should wait for document load